level-2/숫자-블록.js

```diff
@@ -0,0 +1,32 @@
+//https://github.com/codeisneverodd/programmers-coding-test
+//완벽한 정답이 아닙니다.
+//정답 1 - iHoHyeon
+function solution(begin, end) {
+  return new Array(end - begin + 1).fill().map((v, idx) => calc(begin + idx));
```

Comment:
"반영해도 좋고 아니어도 좋아요"

`fill` 메서드 내부에 채울 값을 파라미터로 제공하지 않으면 `undefined`로 값이 채워집니다!
권장되는 방법은 아니니 `fill('')` 혹은 `fill(null)` 은 어떨까요?

```suggestion
  return new Array(end - begin + 1).fill(null).map((v, idx) => calc(begin + idx));
```

[레퍼런스](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/fill)

Reply:
피드백 감사합니다! 반영했어요~